docs/linux_dependencies.md

```diff
@@ -45,48 +45,40 @@ sudo xbps-install -S pkgconf alsa-lib-devel libX11-devel eudev-libudev-devel
 
 ## NixOS
 
-Add a `build.rs` file to your project with the following:
+Add a `build.rs` file to your project containing:
 
 ```rust
+# build.rs
```

Comment:
You've added this comment here, but removed the same from below - I don't think we gain any value from having this comment here, so I'd remove it.

It would also be worth explaining why this comment is needed, because it doesn't seem like adding this should be required for the end user.

Reply:
I would say lets keep these comments. They provide some additional clarity and from experience not everyone reads the text around the code blocks. Sorry I just did quick changes and pushed but haven't reviewed it after myself because I got distracted by something else. I'll clean up after myself, fix the linter errors and request new review.

Reply:
That's fine - I'd say we need to come up with a general policy and have one big cleanup of all of these, but I don't know what the result would be.

Personally, I'd be in favour of not having them, since they mix file content with file names - ideally GitHub flavoured markdown/common mark would let us specify the filename of a code block inline in some standard way, without overloading the code block contents.
However, this PR is not the place to make that decision.

Also, when I wrote
> It would also be worth explaining why this comment is needed, ...

I meant 'why the `build.rs` file is required', but obviously I failed to actually say that 😅